libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Store/SFSDKEventStoreManager.h

```diff
@@ -71,7 +71,7 @@ typedef NSData * _Nullable (^ _Nullable DataDecryptorBlock)(NSData * _Nullable d
  *
  * @return List of event files.
  */
-- (NSArray<SFSDKInstrumentationEvent *> *) eventFiles;
+- (nullable NSArray<NSString *> *)eventFiles;
```

Comment:
Is this a breaking change?

Reply:
No the method has only been in `dev` so far, not released